common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockFetcher.java

```diff
@@ -99,18 +99,30 @@ void createAndStart(String[] blockIds, BlockFetchingListener listener)
    */
   private RetryingBlockFetchListener currentListener;
 
+  private final ErrorHandler errorHandler;
+
   public RetryingBlockFetcher(
```

Comment:
I think the naming become quite misleading as now block pushings are retried by this.
I know the changeset is already huge but please consider renaming this in a follow up PR.

What about `RetryingBlockMover` or `RetryingBlockTransfers`?

Reply:
I commented earlier that we will rename BlockFetchingListener and RetryingBlockFetcher, both of which are reused for block fetch/push, to BlockTransferListener and RetryingBlockTransferor.
I plan to make a quick follow up PR for this naming change to limit the number of files touched in this PR.

Reply:
oh sorry, its my bad I really focused only on the code